### DISCUSSION TITLE
Mover os cards de Criar Scripts e Criar ConsoleApp para Done

### DISCUSSION THREAD

**Discussion by LuanFreitasRibeiro** [andrebaltieri/AgendaEventos#54] [General]
Pessoal, 

O que acham de movermos os cards #17 e #16, para Done? Uma vez que os mesmos já foram revisados e aprovados (inclusive mesclados na branch master/principal).

**Comment by jenrodrigues** [comment]
Com certeza. Ja esta colocando como tarefa na ata dessa semana.

**Reply by jenrodrigues** [reply]
Feito!
